advocacy_docs/partner_docs/PreciselyConnectCDC/05-Usingpreciselyconnectcdc.mdx

```diff
@@ -0,0 +1,309 @@
+---
+title: 'Using'
+description: 'Walkthrough of example usage scenarios'
+---
+
+After you have configured your EDB Postgres Distribution, as stated in the Configuring section, you will be able to then use Precisely CDC's Replication functionality. For the examples in this guide, the replication functionality is demonstrated using EDB Postgres Advanced Server.
+
+- Configure Postgres Distribution as a Source Instance in Connect CDC Director
+  - Setup and configuration of EDB Postgres Advanced Server as a source instance for Connect CDC Director.
+- Configure Postgres Distribution as Target Instance in Connect CDC Director
+  - Setup and configuration of EDB Postgres Advanced Server as target instance to Connect CDC Director.
+- Create Distribution on Connect CDC Director
+  - Define how data will be replicated from source to target in a Model.
+- Save the Model
+  - Save, validate and commit the Model created for Replication.
+- Execute Copy Replication using Connect CDC MonCon
+  - Execute the Copy Replication to copy all the data from the Source Tables to the Target Tables.
+- Create Replication Distribution using Connect CDC Director
+  - Replicate all the changes made on the Source Tables to the Target Tables.
+- Execute Replication using Connect CDC MonCon
+  - Replication distribution will replicate all the changes done on the Source Tables to the Target Tables.
+
+With Precisely Connect CDC you also have the ability to “Transform” your data as you replicate it from a source to target using Data Transformation.  See the section on “Data Transformation” on how to enable this feature.
+
+- Data Transformation
+  - Modify the source data you are distributing or even create or derive the data you distribute to the target system.
+
+
+## Configure Postgres Distribution as a Source Instance in Connect CDC Director
+
+The following steps demonstrate the setup and configuration of EDB Postgres Advanced Server as a source instance for Connect CDC Director. 
+
+1. Open Connect CDC Director.
+
+2. Click `New` option.
+
+3. Right click `Hosts, Servers, Tables` and select `New Host`.  
+
+4. Enter information for the `Host`, `Host` is the machine where Connect CDC Director is running.
+
+![Create New Host](Images/newhost.png)
```

Comment:
Could we crop this image a bit more at the bottom to save space?

Reply:
Done